src/renderer/dx/DxRenderer.cpp

```diff
@@ -79,6 +79,7 @@ DxEngine::DxEngine() :
     _backgroundColor{ 0 },
     _selectionBackground{},
     _haveDeviceResources{ false },
+    _swapChainHandle{ INVALID_HANDLE_VALUE },

```

Comment:
`wil::unique_*` types have default constructors and you don't need to initialize them here.